code/backend/batch/utilities/helpers/ConfigHelper.py

```diff
@@ -77,15 +89,35 @@ def get_active_config_or_default():
 
         if env_helper.LOAD_CONFIG_FROM_BLOB_STORAGE:
             try:
+                default_config = config
+
                 blob_client = AzureBlobStorageClient(
                     container_name=CONFIG_CONTAINER_NAME
                 )
                 config_file = blob_client.download_file("active.json")
-                config = Config(json.loads(config_file))
-            except Exception:
+                config = json.loads(config_file)
+
+                # These properties may not exist in the config file as they are newer
+                config["prompts"]["answering_system_prompt"] = config["prompts"].get(
```

Comment:
I am planning to add tests to cover this scenario once the core of the PR has been reviewed